modules/oadp-upgrade-from-oadp-data-mover-1-2-0.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="upgrade-from-oadp-data-mover-1-2-0_{context}"]
+= Upgrading from OADP 1.2 Technology Preview Data Mover
+
+{oadp-first} 1.2 Data Mover backups *cannot* be restored with OADP 1.3. To prevent a gap in the data protection of your applications, complete the following steps before upgrading to the OADP 1.3 upgrade:
+
+. If your cluster backups are sufficient and Container Storage Interface (CSI) storage is available,
+back up the applications with a CSI backup.
+. If you require off cluster backups:
+.. Back up the applications with a file system backup that uses the `--default-volumes-to-fs-backup=true or backup.spec.defaultVolumesToFsBackup` options.
+.. Back up the applications with your object storage plugins, for example, `velero-plugin-for-aws`.
```

Comment:
Do these need to have specific actions that user needs to do? Overall, this module seems like a procedural module but sounds a bit incomplete.

Reply:
This needs more context, if it is available.

Reply:
It's not presently available.